src/Sarif.Multitool/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -428,6 +427,20 @@ private void Visit(StackFrame frame, string framePointer)
         private void Visit(ThreadFlow threadFlow, string threadFlowPointer)
         {
             Analyze(threadFlow, threadFlowPointer);
+
+            if (threadFlow.Locations != null)
+            {
+                ThreadFlowLocation[] threadFlowLocations = threadFlow.Locations.ToArray();
+                string threadFlowLocationsPointer = threadFlowPointer.AtProperty(SarifPropertyName.Locations);
+
+                for (int i = 0; i < threadFlowLocations.Length; ++i)
+                {
+                    ThreadFlowLocation threadFlowLocation = threadFlowLocations[i];
+                    string threadFlowLocationPointer = threadFlowLocationsPointer.AtIndex(i);
+
+                    Visit(threadFlowLocation, threadFlowLocationPointer);
+                }
+            }
         }
```

Comment:
An existing bug that went unnoticed because I hadn't yet restored a rule that expected to find an issue in a code flow location. I had the code to visit `codeFlow` and loop over its `threadFlow`s, and I had the code to visit a `threadFlowLocation` and loop over its `location`s -- but I was missing the part in the middle where you loop over the `threadFlowLocation`s in a `threadFlow`.